app/services/error_details/create_service.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module ErrorDetails
+  class CreateService < BaseService
+    def initialize(params:, owner:, organization:, integration: nil)
```

Comment:
I think we can remove this constructor since no new code is added inside (compared to the parent)